src/motor_hardware.cc

```diff
@@ -50,6 +50,7 @@ const static uint8_t  I2C_PCF8574_8BIT_ADDR = 0x40; // I2C addresses are 7 bits
 // 17.2328767123 and  gear ratio of 4.29411764706:1
 #define TICKS_PER_RADIAN_ENC_3_STATE (20.50251516)    // used to read more misleading value of (41.0058030317/2)
 #define QTICKS_PER_RADIAN   (ticks_per_radian*4)      // Quadrature ticks makes code more readable later
+#define HIGH_SPEED_RADIANS (1.8) // This is a threshold were we consider we are turning the wheel very fast'
```

Comment:
I had it lower but then rotation with the joystick can do over 1.6 so I put it here.    All this does is pepper the log with messages saying it is happening so I wanted to use a number that seemed high but it is a sort of guess.  I feel they are rotating at some rediculous value because the motor node is crying out that pwm limits are hit and so on.  So we seek notification of really high requests for rotation at high speed.